### DISCUSSION TITLE
Where is the php.ini?

### DISCUSSION THREAD

**Discussion by kauelima21** [asdf-community/asdf-php#113] [Q&A]
How can i configure apache or nginx? I am a newbie and i am starting to learning php without xampp... someone can help me?

**Comment by kauelima21** [comment]
I can't found the php.ini...

**Reply by kiramishima** [reply]
PHP ini is in yout asdf php version installed path. 
Try to run:

```
php -i | grep php.ini
// You'll get a similar output
Configuration File (php.ini) Path => /home/kiramishima/.asdf/installs/php/8.0.6
Loaded Configuration File => /home/kiramishima/.asdf/installs/php/8.0.6/php.ini
```

So, in that path, you can find you php.ini.

For Nginx, you need install php-fpm extension and configure a reverse_proxy in you nginx availables sites/enable (it depends of the version of nginx). For example, in Laravel, they configure a proxy https://laravel.com/docs/9.x/deployment#nginx

**Reply by Wirone** [reply]
You can also use `php --ini` to see this information:

```
$ php --ini

Configuration File (php.ini) Path: /Users/gkorba/.asdf/installs/php/8.3.4
Loaded Configuration File:         (none)
Scan for additional .ini files in: /Users/gkorba/.asdf/installs/php/8.3.4/conf.d
Additional .ini files parsed:      /Users/gkorba/.asdf/installs/php/8.3.4/conf.d/php.ini
```
